src/server/ScriptServer.ts

```diff
@@ -8,6 +8,7 @@ import {
 } from 'vscode-languageserver';
 import { TextDocument } from 'vscode-languageserver-textdocument';
 import { getIndexes, activateIndexes } from './scriptServer/features/indexes';
+import { workspace } from 'vscode';
```

Comment:
You can't use `vscode` module in script server since it's separate process and don't have access to vscode environment. Please test your changes before making PRs.